src/main/java/appeng/client/gui/implementations/GuiCellWorkbench.java

```diff
@@ -47,12 +47,12 @@
 public class GuiCellWorkbench extends GuiUpgradeable
 {
 
-	final ContainerCellWorkbench workbench;
-	final TileCellWorkbench tcw;
+	private final ContainerCellWorkbench workbench;
+	private final TileCellWorkbench tcw;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "tcw" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)